react/ErrorIcon/ErrorIconFilled.svg

```diff
@@ -1 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 101 99.997"><path d="M96.068 30.534c-2.52-5.954-6.124-11.3-10.715-15.89-4.592-4.59-9.938-8.195-15.894-10.714C63.292 1.322 56.743 0 49.996 0c-6.747 0-13.294 1.322-19.46 3.93-5.954 2.52-11.3 6.123-15.89 10.713-4.592 4.59-8.197 9.937-10.716 15.89C1.324 36.7 0 43.25 0 49.997c0 6.75 1.323 13.297 3.93 19.464 2.52 5.954 6.125 11.3 10.715 15.892s9.937 8.195 15.89 10.714c6.167 2.607 12.714 3.93 19.46 3.93 6.75 0 13.297-1.32 19.464-3.93 5.954-2.52 11.3-6.123 15.893-10.714 4.59-4.59 8.196-9.938 10.715-15.892C98.678 63.293 100 56.745 100 49.996c0-6.747-1.323-13.296-3.932-19.462zM46.006 72.492h8.073c.72 0 1.306.585 1.306 1.307v8.02c0 .723-.585 1.308-1.307 1.308h-8.074c-.722 0-1.307-.585-1.307-1.307V73.8c0-.723.584-1.308 1.306-1.308zM44.7 60.946V18.058c0-.722.584-1.307 1.306-1.307h8.02c.724 0 1.31.586 1.31 1.308v42.888c0 .722-.586 1.307-1.31 1.307h-8.02c-.722 0-1.307-.586-1.307-1.307z"/></svg>
+<!-- Generated by IcoMoon.io -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="512" height="512" viewBox="0 0 512 512">
```

Comment:
Should this be `width="16" height="16"`? Right now the filled and unfilled variants are different sizes.